### DISCUSSION TITLE
Stuck wait-close channels: “fee rate delta is zero"

### DISCUSSION THREAD

**Discussion by smashtrick** [lightningnetwork/lnd#9360] [Troubleshooting]
### Background
- LND Version: v0.18.3-beta
- Running on: Raspberry Pi 4
- Backend: Bitcoin Core v24.0
- RTL Version: v0.15.2
- Network: Mainnet

### Problem
I have several wait-close channel that has been stuck for 2+ months. After trying `lncli wallet bumpfee`, the logs show:
[ERR] SWPR: Failed to sweep BudgetInputSet(budget=0.00000330 BTC, deadline=875727, inputs=[934ee7e2c5b61d5a7dc76c23a3216427176274782c11f521fc74c0f2b61b133b:0]): init fee function: fee rate delta is zero

The `pendingchannels` output shows:
"closing_txid": "b3ac2eaec8789895ec8e35f3410e4fed6396bba9ccf1f7a87ad71974b6c3cb65", "limbo_balance": 223797

I tried the following:
- Bumping fees multiple times. I was able to apparently successfully bumb fee "Successfully registered rbf-tx with sweeper” but it still never showed up in mempool. 
- Manually restarting LND.
- Checking `lncli listchaintxns` but the transaction doesn’t show up.
- No `closing_tx_hex` available in the logs.
- Force close and lnd response is: channel is already in the process of being force closed

### What I Need
- How can I manually broadcast or force this transaction?
- Should I consider using SCB recovery, or is there another way to recover the limbo balance?

**Comment by ziggie1984** [comment]
Can you verify that the channel you are trying to close exists onchain?

Moreover when trying to bump fees via the sweeper please read the sweeper doc thoroughly:

https://github.com/lightningnetwork/lnd/blob/master/sweep/README.md

**Comment by cocktailsk** [comment]
HI [smashtrick]

were you able to solve this? I have the same issue with a force closed channel that had HTLC's. LND did recovered some funds but there is still one output pending. 
   | 2025-07-07 07:31:43.449 [ERR] SWPR: Failed to sweep BudgetInputSet(budget=0.00998460 BTC, deadline=905427, inputs=[
lightning_lnd_1        | a9c786f914f9ffde990e6c1213112701dac9642694f5f3e34c5bacb2420fe435:5 (HtlcOfferedTimeoutSecondLevelInputConfirmed)

the sweep tx was not broadcasted to the network so no reason to bump fees.
